src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
```

Comment:
I know that many people prefer shorthand, smth like 'cn'

Reply:
I don't think that naming classNames like cn will shorten my code or even will make it more readable, moreover IMHO classNames is more obvious and it can be easily found in npm doc, so even if reader isn't familliar with cn, he/she can easily find out more about it. So I think it doesn't matter how do I name this, cn or classNames

Reply:
hey @davinnchii
to be honest, using `cn` or `cx` as shorthand for the classnames library is quite prevalent in real projects. Our primary objective is to guide you in becoming a proficient developer, well-versed in the industry's standards and best practices. If you ever have any questions or need assistance, please don't hesitate to reach out to me on Slack.